Import url in example is wrong

Your example code in the readme points to a non-existing repository:
Instead of
`uses: wzieba/App-Center-action@v1.0.0`
it should be
`uses: wzieba/AppCenter-Github-Action@v1.0.0`
Which then points to this source: https://api.github.com/repos/wzieba/AppCenter-Github-Action/tarball/v1.0.0

Otherwise I get this error in my action:
```
Download action repository 'wzieba/App-Center-action@v1.0.0'
##[warning]Failed to download action 'https://api.github.com/repos/wzieba/App-Center-action/tarball/v1.0.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 14.427 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/wzieba/App-Center-action/tarball/v1.0.0'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 10.353 seconds before retry.
##[error]Response status code does not indicate success: 404 (Not Found).
```

--- a/README.md
+++ b/README.md
@@ -54,7 +54,7 @@ jobs:
     - name: build release
       run: ./gradlew assembleRelease
     - name: upload artefact to App Center
-      uses: wzieba/App-Center-action@v1.0.0
+      uses: wzieba/AppCenter-Github-Action@v1.0.0
       with:
         appName: wzieba/Sample-App
         token: ${{secrets.APP_CENTER_TOKEN}}

